app/views/content/shared/quotes/_quote_career_change_jack.html.erb

```diff
@@ -1,7 +1,5 @@
-<div class="quote-shiulee">
- <%= render Content::QuoteComponent.new(
-      text: "I was stuck in the corporate rat race in London and just wanted to do something worthwhile.",
-      name: "Jack, French and German teacher",
-      large: false
-    ) %>
-</div>
+<%= render Content::QuoteComponent.new(
+  text: "I was stuck in the corporate rat race in London and just wanted to do something worthwhile.",
+  name: "Jack, French and German teacher",
+  large: false
```

Comment:
Looks like the default value for `large` is `false` on QuoteComponent, we could safely remove any `large: false` to tidy up further